demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -58,6 +58,9 @@ def __init__(
         self.code_type = self.get_code_type()
         self.yml_docker_image = self.get_docker_image_from_yml()
         self.from_version = self.yml_file.get("fromversion", "0")
+        self.is_pack_xsoar_supported = (
+            get_pack_metadata(yml_file_path).get("support", "xsoar").lower() == "xsoar"
```

Comment:
should the default be xsoar? I imagine we would be more careful with this than customers will be. IMO should change default to cimmunity

Reply:
If no info is found than validate as it was xsoar supported - sounds the safest.

Reply:
are we able to assume that customers will have this field set on their own cicd pipelines? @dorschw maybe you know if we can rely on that?

Reply:
Yes, there's a validate to ensure this field exists in the pack_metadata